src/notebook/plugin.ts

```diff
@@ -142,7 +146,11 @@ const plugin: JupyterLabPlugin<INotebookTracker> = {
 /**
  * Activate the notebook handler extension.
  */
-function activateNotebookHandler(app: JupyterLab, registry: IDocumentRegistry, services: IServiceManager, rendermime: IRenderMime, clipboard: IClipboard, mainMenu: IMainMenu, palette: ICommandPalette, inspector: IInspector, renderer: NotebookPanel.IRenderer, state: IStateDB, layout: ILayoutRestorer): INotebookTracker {
+function activateNotebookHandler(app: JupyterLab, registry: IDocumentRegistry, services: IServiceManager, rendermime: IRenderMime, clipboard: IClipboard, mainMenu: IMainMenu, palette: ICommandPalette, inspector: IInspector, editorServices: IEditorServices, state: IStateDB, layout: ILayoutRestorer): INotebookTracker {
+
+  const notebookRenderer = new Notebook.Renderer({ editorServices });
```

Comment:
Okay, then I will make the default `IRenderer` plugins use `IEditorServices` only, but allow Monaco to provide the more advanced `IRenderer`.